src/powershell/install-chocolatey.ps1

```diff
@@ -0,0 +1,55 @@
+# Install and configure Chocolatey
+# See: https://community.chocolatey.org/
+
+function Install-Chocolatey {
+    # Download and execute the Chocolatey install script
+    iex ((New-Object System.Net.WebClient).DownloadString("https://community.chocolatey.org/install.ps1"))
```

Comment:
This should be the same functionality but consistent with other PowerShell in this project.

```suggestion
    Invoke-Expression ((New-Object System.Net.WebClient).DownloadString("https://community.chocolatey.org/install.ps1"))
```

Reply:
@mcdonnnj resolved in [ede1800](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/ede1800375ef4563e799d0d4d1f001cda9d49244)

Sorry, I'm aware I didn't update the commit comment. I accidentally hit enter before I can update the comment first